superdesk/wsgi.py

```diff
@@ -0,0 +1,11 @@
+"""WSGI entrypoint.
+
+Usage via Procfile:
+
+    rest: gunicorn -c pythong:superdesk.gunicorn_config superdesk.wsgi
+
+"""
+
+from superdesk.factory import get_app
+
+application = get_app()
```

Comment:
I think `if __name__ == '__main__':` won't work, but not sure, but if we can somehow, then sounds good